### PR TITLE
Role Agnostics: IsOmniscientRole

### PR DESCRIPTION
**Pull request by TimGoll**
This new role variable moves yet another traitor feature to the role base. If `isOmniscientRole` is set to true, the role is able to see the haste mode timer as well as missing in action players.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - equipment specific settings can be found in the 'edit equipment' menu
 - Added icon to the magneto stick
 - Added the function `AddToSettingsMenu` to both `SWEP` and `ITEM` to add settings to the equipment menu
+- Added the role flag `.isOmniscientRole`; if set to true the role is able to see missing in action players and the haste mode time
```
Comment (ZenBre4ker) [inline CHANGELOG.md:15]:
```suggestion
- Added the role flag `.isOmniscientRole`, if set to true the role is able to see missing in action players and the haste mode time
```

**Review by ZenBre4ker — APPROVED** [review]
I like the way these role agnostics are heading. <3

**Review by Histalek — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (TimGoll) [reply CHANGELOG.md:15]: I did not see this when merging. However I think that a `;` is more fitting anyway

Comment (ZenBre4ker) [reply CHANGELOG.md:15]: well, we nver use semicolons, but it's okay 😄

Comment (TimGoll) [reply CHANGELOG.md:15]: I actually use them quite a lot in in-code comments

**Pull request merged**